src/json_reporter.cc

```diff
@@ -162,11 +162,11 @@ bool JSONReporter::ReportContext(const Context& context) {
 #else
   const char build_type[] = "debug";
 #endif
-  out << indent << FormatKV("library_build_type", build_type) << "\n";
+  out << indent << FormatKV("library_build_type", build_type) << ",\n";
```

Comment:
Unfortunately JSON does not allow trailing commas, so this would still result in invalid JSON files.